src/components/widgets/AutocompleteWidget/AutocompleteWidget.tsx

```diff
@@ -4,202 +4,208 @@ import ReactDOM from "react-dom";
 import { OlsApi } from "../../../api/OlsApi";
 import { EuiComboBoxOptionOption } from "@elastic/eui/src/components/combo_box/types";
 import {
-    EuiComboBox,
-    euiPaletteColorBlindBehindText,
-    euiPaletteColorBlind,
-    EuiHighlight,
-    EuiHealth,
-    EuiProvider
+  EuiComboBox,
+  euiPaletteColorBlindBehindText,
+  euiPaletteColorBlind,
+  EuiHighlight,
+  EuiHealth,
+  EuiProvider
 } from "@elastic/eui";
-import {QueryClient, QueryClientProvider, useQuery} from "react-query";
-import {AutocompleteWidgetProps} from "../../../app/types";
+import { QueryClient, QueryClientProvider, useQuery } from "react-query";
+import { AutocompleteWidgetProps } from "../../../app/types";
 import { BreadcrumbPresentation } from "../MetadataWidget/BreadcrumbWidget/BreadcrumbPresentation";
+import { BreadcrumbWidget } from "../MetadataWidget";
```

Comment:
Is this needed? Or is the BreadcrumbPresentation in line 16 enough? I would guess we could skip this integration and save some requests.